compute/admin_guide/install/deploy-defender/redeploy-defender.adoc

```diff
@@ -20,9 +20,16 @@ $ <PLATFORM>/twistcli defender export kubernetes \
     --cluster-address twistlock-console
 ----
 
-. Apply the https://kubernetes.io/docs/concepts/cluster-administration/manage-deployment/#in-place-updates-of-resources[in-place updates] to your `Defender` resources.
+. Delete the old Defenders using your old daemonset config file:
```

Comment:
Please confirm that with QA also. The main reason is that does deleting the Defenders contradict the next step where we talk about "in-place updates".

Reply:
@Pubs-MV validated on the ticket.